packages/astro/package.json

```diff
@@ -17,6 +17,13 @@
     "node": ">=18.14.1"
   },
   "type": "module",
+  "files": [
+    "cjs",
+    "esm",
+    "types",
+    "types-ts3.8",
```

Comment:
Astro doesn't have `integration` and `types-ts3.8` on npm. Also I think we should also add README.md files. https://www.npmjs.com/package/@sentry/astro?activeTab=code

Reply:
Oh you're right. I believe we had it at some point, so we probably forgot to remove it previously.

Reply:
We don't need to explicitly add READMEs. They are included by npm by default just like the license and the package json itself: https://docs.npmjs.com/cli/v10/configuring-npm/package-json#files